tests/integration/test_store.py

```diff
@@ -1320,30 +1320,30 @@ def test_kafka_ssl(relay_with_processing):
 
 
 @pytest.mark.parametrize(
-    "normalize",
+    "normalization_level",
     ["disabled", "default", "full"],
 )
 def test_relay_normalization(
     mini_sentry,
     relay,
-    normalize,
+    normalization_level,
 ):
     project_id = 42
     mini_sentry.add_basic_project_config(project_id)
     relay = relay(
         upstream=mini_sentry,
-        options={"processing": {"normalize": normalize}},
+        options={"normalization": {"level": normalization_level}},
```

Comment:
The test should have failed with invalid config -- https://github.com/getsentry/relay/issues/3429.